pallets/pips/src/lib.rs

```diff
@@ -446,12 +457,20 @@ decl_storage! {
         /// Maps PIPs to the block at which they will be executed, if any.
         pub PipToSchedule get(fn pip_to_schedule): map hasher(twox_64_concat) PipId => Option<T::BlockNumber>;
 
+        /// A live priority queue (lowest priority at index 0)
+        /// of pending non-cooling PIPs up to the active limit.
```

Comment:
non-cooling bit can be removed.

Reply:
Perfect; thanks for reminding me of this. I went ahead and did a proper grep for `cool` and fixed the docs more broadly.